### PR TITLE
Add Transifex and German translations

### PR DESCRIPTION
**Pull request by FinalAngel**
* Configured Transifex
* Generated German translations
* Pulled German translations form Transifex
* Update changelog

### CODE REVIEW

File: aldryn_forms/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # -*- coding: utf-8 -*-
-__version__ = '2.1.1'
+__version__ = '2.1.2'
```
Comment (czpython) [inline aldryn_forms/__init__.py]: Please follow the convention to create a separate commit that **only** contains the version bump with a message like so:

> Version bump to x.y.z

version-bump

**Comment by FinalAngel** [comment]
@czpython will do so, have not yet finished this (translations still ongoing)

File: aldryn_forms/__init__.py (hunk 1 above)
Comment (FinalAngel) [inline aldryn_forms/__init__.py]: done :)

File: aldryn_forms/forms.py [hunk 2]
```diff
@@ -59,13 +59,19 @@ def clean(self, *args, **kwargs):
 
         if self.max_width and width > self.max_width:
             raise forms.ValidationError(
-                ugettext('Image width must be under %s pixels. '
-                         'Current width is %s pixels.') % (self.max_width, width))
+                ugettext('Image width must be under %(max_size)s pixels. '
```
Comment (czpython) [inline aldryn_forms/forms.py]: indentation is a bit off here, maybe do:

```python
message = ugettext(
    'Image width must be under %(max_size)s pixels.'
    'Current width is %(actual_size)s pixels.'
) % {'max_size': self.max_width, 'actual_size': width}
```

File: aldryn_forms/forms.py [hunk 3]
```diff
@@ -59,13 +59,19 @@ def clean(self, *args, **kwargs):
...
+                         'Current width is %(actual_size)s pixels.') % {
+                    'max_size': self.max_width,
+                    'actual_size': width,
+                }
 
         if self.max_height and height > self.max_height:
             raise forms.ValidationError(
-                ugettext('Image height must be under %s pixels. '
-                         'Current height is %s pixels.') % (self.max_height, height))
+                ugettext('Image height must be under %(max_size)s pixels. '
```
Comment (czpython) [inline aldryn_forms/forms.py]: same as above

**Comment by czpython** [comment]
@FinalAngel Ok to merge after addressing comments.

**Pull request merged**